lib/ex_stun/message.ex

```diff
@@ -370,4 +397,25 @@ defmodule ExSTUN.Message do
   end
 
   defp strip_padding(_data, _padding_len), do: {:error, :malformed_attr_padding}
+
+  defp get_message_integrity(msg) do
+    case get_attribute(msg, MessageIntegrity) do
+      nil -> {:error, :no_message_integrity}
+      other -> other
```

Comment:
Not sure how we should treat incorrect attribute? Should we just ignore it and return `{:error, :no_message_integrity}`?